tag/key.go

```diff
@@ -21,7 +21,7 @@ type Key struct {
 }
 
 // NewKey creates or retrieves a string key identified by name.
-// Calling NewKey consequently with the same name returns the same key.
+// Calling NewKey (or MustKey) with the same name returns the same key.
```

Comment:
Looks like a stray comment, the `MustKey` or do we already have `MustKey`? And I believe we were talking of not exposing MustKey, right?